packages/ag-charts-enterprise/src/features/annotations/axisButton.css

```diff
@@ -0,0 +1,31 @@
+.annotations-axis-button {
+    position: absolute;
+    left: 0px;
+    top: 0px;
+    user-select: none;
+    font:
+        15px Verdana,
+        sans-serif;
+    overflow: hidden;
+    white-space: nowrap;
+    z-index: 99998;
+    box-sizing: border-box;
+}
+
+.annotations-axis-button button {
+    font-size: 15px;
+    background: rgb(255, 255, 255);
+    border: 1px solid rgb(71, 71, 71);
+    border-radius: 10px;
+    padding: 0 4px;
+    color: rgb(71, 71, 71);
+}
+
+.annotations-axis-button button:hover {
+    background: rgba(71, 71, 71, 0.8);
+    color: rgb(255, 255, 255);
+}
+
+.annotations-axis-button-hidden {
+    top: -10000px !important;
```

Comment:
I think `display: none` or `visibility: hidden` would be better, if either of those work for your situation.